### DISCUSSION TITLE
Running proxy mode cause issue with connecting to external server

### DISCUSSION THREAD

**Discussion by waqasraz** [kubenetworks/kubevpn#353] [Q&A]
Hello, and first, thank you for maintaining this project. 
So, our service is built with node. After I ran the kubevpn command to start proxy mode to start local development, I ran `npm run start`. Since it uses a VPN, the network does not recognize an external server, and it gives a timeout. Do you have any idea how to provide that information?

So if I stop the proxy mode and run 'npm run start' everything connected successfully
```
azure.confluent.cloud:9092"},"retryCount":5,"retryTime":10822}
stack: KafkaJSNonRetriableError
  Caused by: KafkaJSConnectionError: Connection timeout
```

**Comment by waqasraz** [comment]
sorry I think I was doing something wrong let me double check

**Reply by wencaiwulue** [reply]
@waqasraz hello, do you solve this problem?

**Reply by waqasraz** [reply]
The issue was on our side. The firewall was blocking the connection.

 Any idea on when I run on Windows native kubevpn connect, it just gets stuck there no error, no log, nothing.

**Reply by wencaiwulue** [reply]
> The issue was on our side. The firewall was blocking the connection.
> 
> Any idea on when I run on Windows native kubevpn connect, it just gets stuck there no error, no log, nothing.

Can you use kubectl to access api-server on Windows ? to verify connection is ok or not ?
